migration/iD11/oracle/202310061103_IDEMPIERE-3989.sql

```diff
@@ -0,0 +1,73 @@
+-- IDEMPIERE-3989
+SELECT register_migration_script('202310061103_IDEMPIERE-3989.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Oct 6, 2023, 11:03:05 AM BRT
+INSERT INTO AD_ViewColumn (AD_Client_ID,AD_Org_ID,AD_ViewColumn_ID,AD_ViewColumn_UU,Created,CreatedBy,EntityType,IsActive,Updated,UpdatedBy,AD_ViewComponent_ID,ColumnName,ColumnSQL,SeqNo) VALUES (0,0,nextidfunc(200086,'N'),'2cca7f60-b49c-414d-bf75-7f6aad3d39a4',TO_TIMESTAMP('2023-10-06 11:03:05','YYYY-MM-DD HH24:MI:SS'),100,'U','Y',TO_TIMESTAMP('2023-10-06 11:03:05','YYYY-MM-DD HH24:MI:SS'),100,200008,'entitytype','f.entitytype',720)
```

Comment:
the entity type of the 5 new records on AD_ViewColumn are wrong, must be 'D' instead of 'U'